mods/ra/tilesets/interior.yaml

```diff
@@ -1181,3 +1181,192 @@ Templates:
 		Categories: Floor
 		Tiles:
 			0: Clear
+	Template@291:
+		Id: 291
```

Comment:
They should be. I used `Sarin Gas 2: Down Under` as testcase (`Mousetrap` doesn't have those walls?).